samples/matter/light_switch/Kconfig.sysbuild

```diff
@@ -4,34 +4,34 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+#### Radio core selection
 config NRF_DEFAULT_IPC_RADIO
 	default y
 
+# Enable IEEE802.15.4 serialization to network core
 config NETCORE_IPC_RADIO_IEEE802154
-	default y if BOARD_NRF5340DK_NRF5340_CPUAPP
+	default y if SOC_SERIES_NRF53X
 
+# Enable Bluetooth serialization to network core
 config NETCORE_IPC_RADIO_BT_HCI_IPC
-	default y if (BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP)
-
-config MATTER_FACTORY_DATA
-	default y if !BOARD_NRF21540DK
+	default y if SOC_SERIES_NRF53X
```

Comment:
Ditto, do we want to enable BT_HCI_IPC for nRF5340DK and IEEE802154 for nRF7002DK?

Reply:
Right, will repair that